config-github.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AWS_ACCESS_KEY_ID": "Description",
+  "AWS_SECRET_ACCESS_KEY": "Description",
+  "ENC_KEY": "Description",
+  "PROD_ENC_KEY": "Description"
+}
```

Comment:
this file is meant to live in the project that installs this library; it'll work in much the same way that `config-{stage}.yml` and `config-required.json` work. Could this be placed next to `config-development.yml` in `example_project`, and the code modified to always try and read `config-github.json` from the current working directory? So if you're in the wrong directory, it should fail of course.

These config files belong in the root of the django project, alongside manage.py

Reply:
This should already be attempting to read from the current working directory. I did move the file to the `example_project`, but the code should already be reading from the current dir.